config/opal_setup_cc.m4

```diff
@@ -157,9 +151,26 @@ AC_DEFUN([OPAL_SETUP_CC],[
 
         # Get the correct result for C11 support flags now that the compiler flags have
         # changed
-        OPAL_PROG_CC_C11_HELPER([],[],[])
+        OPAL_PROG_CC_C11_HELPER([], [], [])
```

Comment:
Do you know what this is for?  We only get here if we have already determined that there is no C11 support, so we check for C99 support.  ...but then we check for C11 support again, with no CLI options.

Are we just checking to see if the C99 flags also enable *some* C11 functionality?  If so, the comment above this line is misleading, at best.

Reply:
My guess is we want to make sure all C11 macros are set to zero.

@hjelmn can you please comment on that ?

Reply:
That check is there because the C99 support check may add additional flags (for example ```--std=gnu99```) which may change the results. This is entirely defensive. Example (have not seen this happen), without a flag: _Generic = 1, with ```--std=gn99``` _Generic = 0.

Reply:
It may never happen but that was the original reason. The line dates to the earliest versions of the C11 checks.